async-embedded/Cargo.toml

```
@@ -13,8 +13,7 @@ pin-utils = "0.1.0"
 typenum = "1.12.0"
 
 [target.'cfg(target_arch = "arm")'.dependencies]
-cortex-m = "0.6.2"
-cortex-m-udf = { path = "../cortex-m-udf" }
+cortex-m = "0.6"
```

```suggestion
cortex-m = "0.6.3"
```
`udf` arrived only in v0.6.3, any other version will not work.

---

Fixed, thanks!